packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec

```diff
@@ -116,6 +116,12 @@ install -Dp -m0644 %{SOURCE1} %{buildroot}%{?scl:%_root_sysconfdir}%{!?scl:%_sys
 exit 0
 
 %changelog
+* Fri Mar 08 2019 Timo Goebel <mail@timogoebel.name> - 6.0.0-1
```

Comment:
you got the changelog history wrong ;)

Reply:
Darn, that could have been avoided.

🚂 

Fixed.